dbt/models/reporting/docs.md

```diff
@@ -102,16 +102,18 @@ Assessed and market values by PIN and year, for each assessment stage.
 
 The assessment stages are:
 
-1. `mailed` - Values initially mailed by the Assessor
-2. `certified` - Values after the Assessor has finished processing appeals
-2. `board` - Values after the Board of Review has finished their appeals
+1. `PRE-MAILED` - Provisional values that are slated to be mailed by the
+   Assessor once first-pass desk review completes
+2. `MAILED` - Values initially mailed by the Assessor
+3. `ASSESSOR PRE-CERTIFIED` - Provisional values that are slated to be set by
+   the Assessor once appeals are finished
+4. `ASSESSOR CERTIFIED` - Values after the Assessor has finished processing
+   appeals
+5. `BOARD CERTIFIED` - Values after the Board of Review has finished their
+   appeals
 
 ### Assumptions
 
-- Taking an arbitrary value by 14-digit PIN and year is sufficient for accurate
-  values. We do this because even given the criteria to de-dupe `asmt_all`,
-  we still end up with duplicates by PIN and year.
```

Comment:
Since `reporting.vw_pin_value_long` now pulls from `default.vw_pin_value`, it no longer uses `ARBITRARY` operators (`default.vw_pin_value` does that instead), so these docs no longer apply. Note that `default.vw_pin_value`, which _does_ still use `ARBITRARY`, [already documents the use of the operator](https://github.com/ccao-data/data-architecture/blob/master/dbt/models/default/docs.md#vw_pin_value).